src/components/TokenTotalLineItem.tsx

```diff
@@ -63,9 +77,9 @@ export default function TokenTotalLineItem({
         }
         amount={
           <TokenDisplay
-            amount={tokenAmount}
+            amount={tokenAmount.plus(newSendScreen ? feeInToken ?? 0 : 0)}
             tokenId={tokenId}
-            showLocalAmount={false}
+            showLocalAmount={!showLocalAmount}
```

Comment:
This is confusing is it correct that the show showLocalAmount prop takes the opposite of showLocalAmount?

Reply:
Yeah this part was a bit confusing but the logic is correct. There are two totals being displayed here, the actual total, and the converted total. We want these to be opposite (i.e. if one is in local amount, the other should be in crypto amount). The `showLocalAmount` param is for the Total, and this part of code is for the exchange rate where we want to do the opposite.

I could create an additional variable `const showLocalAmountForConversion = !showLocalAmount` to make it more clear, or I could add a comment if that would be sufficient.

Reply:
I added in a comment that explains this

Reply:
It seems like this component displays both a local currency amount and a fiat amount, and it looks like `showLocalAmount` actually means whether local amount is displayed in the first right row? Maybe this can be named better. Also, it looks like the new props added to this component are essentially only sent on the SendConfirmation page and seems to be derived from `newSendScreen` feature gate. If we want to keep this component generic enough, we should having a param called `newSendScreen`, which is specific to a usage of this component. Maybe its just simpler to make a new version of TokenTotalLineItem for the new send confirmation screen?

Reply:
@satish-ravi I just pushed some changes to clean up TokenTotalLineItem. There are really only two pieces that are new, the `showApprox` and the different exchange rate string. It seems like it would be a bit overkill to make a different component for this since other than 2 lines they are exactly the same. 

I also think it makes sense to have a param for showing local amount (I renamed it to `showLocalAmountForTotal` to clear up confusion), since then when using this component you can decide which order you want (fiat for total and crypto below or vice versa). 

However if these changes make it confusing or there is a better way to implement a new component without as much duplication let me know and i can do that.